Basic Programs/Flipcoin.html

```diff
@@ -0,0 +1,30 @@
+<html>
+<head>
+    <h2>Number of times to flip<span id = 'Giveninput'></span></h2>
+    <h2>Number of heads <span id = 'heads'></span></h2>
+    <h2>Number of tails <span id = 'tails'></span></h2>
+</head>
+<body>
+    <label for="input">Number of times to flip</label>
+    <input type = "text" id = "input"/>
+    <button id = "submit" onclick="myFunction()">Submit</button>
+    <script>
+        function myFunction(){
+            var randomValue, heads = 0, tails = 0;
+        //Prompt -- if you want the user to input a value before entering a page.
+        //const NoOfTimesToFlip = prompt('Please enter the number of times to Flip Coin');
+        const NoOfTimesToFlip = document.getElementById("input").value;
+        for(var index = 0; index < NoOfTimesToFlip; index++){
+            randomValue = Math.random();
```

Comment:
Use Math.random()directly instead of storing it in a variable as it is being used only once